build_projects/shared-build-targets-utils/Utils/DebPackageCreator.cs

```diff
@@ -111,9 +111,8 @@ private void CreateEmptyDebianLayout(DebianLayoutDirectories layoutDirectories)
             foreach (var variable in debianConfigVariables)
             {
                 var variableToken = $"%{variable.Key}%";
-                debianConfigFileContents.Replace(variableToken, variable.Value);
+                debianConfigFileContents = debianConfigFileContents.Replace(variableToken, variable.Value);
```

Comment:
You could use a string builder here and then you wouldn't have to keep doing this ugly assignment.